store/resource.js

```diff
@@ -42,7 +42,7 @@ class Resource {
   retrieve(...scopes) {
     const { request } = this.options
     if (this.id == null) {
-      return request(this.url).then((data) => {
+      return request(this.url).then(({ data }) => {
```

Comment:
Why do you do this? Do you have any place to check headers or status?

Reply:
Our [store request](https://github.com/oacore/dashboard/blob/master/store/root.js#L22) works like that. So I just wanted to unify it.